### ISSUE TITLE
0.7.15 OB Bug

### ISSUE THREAD

**Issue by gilbox** [moreartyjs/moreartyjs#77]
:-1: Bad news -- There's a bug related to the OB changes in `0.7.15` that I'm still trying to fix.

:+1: Good news -- the bug seems to be related to using `this.setState(..)` in a Morearty-managed component which isn't something we should be doing anyway. I thought that my use-case was special and using setState was OK, but after removing all `setState` calls and re-evaluating the code I don't think it was necessarily a good case to use `setState`.

**Comment by Tvaroh** [comment]
Gil, do you have a failing test case for this?

**Comment by Tvaroh** [comment]
@gilbox, any updates on this problem? Can I be helpful?

**Comment by gilbox** [comment]
Been too busy to look at it :-/ .... hopefully soon. I think what we need is to construct a test that uses `this#setState` and/or `this#forceUpdate` and breaks the OB update mechanism. The bug I encountered resulted in a missed `render()`. However, it was too complicated to really nail down the reason for it. All I know is that it worked in 0.7.14 but not 15. However, it did work in 15 after I refactored to use bindings and no setState/forceUpdate stuff.
